src/app/pages/LiveRadioPage/__snapshots__/index.test.tsx.snap

```diff
@@ -237,10 +237,10 @@ exports[`Radio Page Main should match snapshot for Canonical 1`] = `
   position: relative;
   width: 100%;
   margin: 0 0 1.5rem 0;
+  margin: 0 0 0.5rem 0;
```

Comment:
This is coming from the `audioFigure` styling - wonder if this should only be included on live pages, or does it matter - it will be overridden by the margin on the line above... right?